pkg/startup/v7/startup.go

```diff
@@ -133,16 +134,18 @@ func (s *startup) writeFiles(role api.AgentPoolProfileRole, w writers.Writer, ho
 			filepath = strings.TrimPrefix(filepath, "worker")
 		}
 
-		b, err := template.Template(filepath, tmpl, map[string]interface{}{
-			"Deref": func(pi *int) int { return *pi },
-		}, map[string]interface{}{
-			"ContainerService": s.cs,
-			"Config":           &s.cs.Config,
-			"Derived":          derived,
-			"Role":             role,
-			"Hostname":         hostname,
-			"DomainName":       domainname,
-		})
+		b, err := template.Template(filepath, tmpl,
+			map[string]interface{}{
```

Comment:
Can we revert the whitespace change to reduce the size of this diff? I had a hard time figuring out what changed here.

Reply:
a lot of the whitespace comes from gofmt, but you can use this: https://github.com/openshift/openshift-azure/pull/1817/files?w=1